ligo/Makefile

```diff
@@ -10,7 +10,7 @@ BUILD = $(LIGO) compile-contract --syntax cameligo
 # Where to put build files
 OUT ?= out
 
-.PHONY: all clean test
+.PHONY: all clean build test
```

Comment:
You added `build` here, but there is no `build` target in this file :confused:

Reply:
:ffuuu: